src/surfaces/cardActions/cardActions.tsx

```diff
@@ -0,0 +1,17 @@
+import MUICardActions from '@material-ui/core/CardActions';
+import React from 'react';
+
+function CardActions(props) {
+    const {
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICardActions
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...otherProps}
+        />
+    );
```

Comment:
Same here; I think we can skip destructuring `props` into `otherProps` and just spread `props` directly in the return JSX.

Reply:
Fixed.

Reply:
Looks good.  Resolving.